Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_botiboti/src/main.rs

```diff
@@ -0,0 +1,68 @@
+use num_bigint::{BigUint, ToBigUint};
+use std::io;
+
+// convert input string to BigUint
+fn read_parse(inp: String) -> BigUint {
+    let age: BigUint = inp.trim().parse().expect("Not a positive integer.");
+    age
+}
+
+// turn age in years to days
+fn calc_days(age: BigUint) -> BigUint {
+    let days_in_a_year: BigUint = 365_i32
+        .to_biguint()
+        .expect("Something went wrong with the conversion 365: i32 -> BigUint.");
+    age * days_in_a_year
+}
+
+fn main() {
+    println!("Enter age.");
+
+    let mut age = String::new();
+
+    io::stdin()
+        .read_line(&mut age)
+        .expect("Failed to read line.");
+
+    let age: BigUint = read_parse(age);
+    println!("You entered: {}.", age);
+    println!("That's roughly {} days.", calc_days(age));
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn an_ordinary_human_age() {
+        let result = read_parse(String::from("50"));
+        let expectation: BigUint = 50_i32
```

Comment:
You can use `.unwrap()` in tests.
You can also [use `BigUint::from(42_u32)`](https://docs.rs/num-bigint/latest/num_bigint/struct.BigUint.html#impl-From%3Cu32%3E-for-BigUint).